packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -226,6 +249,9 @@ class OpPerfTelemetry {
 					latencyStats.opPerfData.lengthInboundQueue = this.deltaManager.inbound.length;
 				}
 			}
+			if (typeof message.contents === "string") {
```

Comment:
If possible, would be great to assert all possible types here (I assume it's undefined or string).
Mostly thinking about a case if we ever implement binary payloads here and it switches from string to array - would be nice to capture it right away and change this code.
But it's hard or risky, then ok to skip

Reply:
@vladsud , Done.